packages/shared-components/src/components/onto-dropdown/onto-dropdown.tsx

```diff
@@ -91,35 +91,43 @@ export class OntoDropdown {
 
     render() {
         return (
-            <div class={`onto-dropdown ${this.open ? 'open' : 'closed'}`}
-                 title={this.dropdownButtonTooltip ? this.dropdownButtonTooltip : TranslationService.translate(this.dropdownButtonTooltipLabelKey)}>
-                <button class="onto-dropdown-button"
-                        onClick={() => this.toggleComponent()}>
-                    <span class={'button-icon ' + this.iconClass}></span>
-                    <span class='button-name'>
+          <div class={`onto-dropdown ${this.open ? 'open' : 'closed'}`}>
+            <onto-tooltip
```

Comment:
I know that we implemented it this way for the YASGUI, but having the tooltip as a standalone component by itself requires to wrap it around the element on which we need to show the tooltip.  And I need to say that I'm not quite happy with this approach, because it looks quite verbose. I think that we should reconsider it more like an attribute directive in angular instead.

Reply:
I didn’t find a way to do it.

Reply:
![image](https://github.com/user-attachments/assets/1f8595eb-f2b3-42c9-97d0-74170f8b241d)

Reply:
I'll push a branch so that we can discuss on it

Reply:
I have pushed an example component here.

Reply:
Done